src/mnt_setup_fnc.cpp

```diff
@@ -8,59 +8,88 @@
 std::string const DATADIR = "/storage/epp2/phshgg/DVTuples__v23/";
 
 struct path_data {
-	std::string const DATADIR;
-	std::string sqrts,
+	std::string const DATADIR,
+	sqrts,
 	stripping,
 	year,
 	polarity,
 	stream;
 };
 
-void make_histogram(struct path_data path_in, std::string hist_name, std::string hist_setup, int const nbins, float const xmin, float const xmax, std::string expression, THStack *hist_stack) {
+struct input {

```

Comment:
Would the intention be clearer if you called this `plot_config` rather than `input`?